documentation/developers/development-environment.md

```diff
@@ -55,3 +56,14 @@ pip install pyzmq
 ```
 
 You will have to start Jukebox core application and the WebUI separately. The MPD usually runs as a service.
+
+### Using WSL
+
+You can also use WSL on Windows 10 or 11. This section describes how to use WSL with Visual Studio Code.
+
+1. Install a Debian or Ubuntu image from Microsoft Store
```

Comment:
I think you need to install wsl as well. I would provide a link here:
https://learn.microsoft.com/en-us/windows/wsl/install